pkg/sqlite/sql.go

```diff
@@ -108,7 +108,8 @@ func getSearchBinding(columns []string, q string, not bool) (string, []interface
 	}
 
 	queryWords := strings.Split(q, " ")
-	trimmedQuery := strings.Trim(q, "\"")
+	trimmedQuery := strings.Trim(q, " \"")
```

Comment:
This will result in inconsistent behaviour:

Search `aaa bbb`, searches `aaa` or `bbb`.
Search ` aaa bbb` (leading whitespace), will search `aaa bbb` because `trimmedQuery != q`.

If we're improving this code, then we should address another issue:
Search `aaa  bbb` (double-space), will return everything, due to one of the `queryWords` being `""`.

Suggest the following:
* trim whitespace first
* if string starts and ends with `"`, then search exact query
* otherwise, split using `" "`, remove any empty strings, and search for any of the words

This also needs to have unit tests added.

Reply:
ok I changed the way to deal with double space and whitespace. And I added unit tests.